go/cmd/vtbench/vtbench.go

```diff
@@ -18,16 +18,16 @@ package main
 
 import (
 	"context"
-	"flag"
 	"fmt"
 	"strings"
 	"time"
 
 	"github.com/spf13/pflag"
 
+	"vitess.io/vitess/go/vt/grpccommon"
+
```

Comment:
We don't really need a space here, but I don't care as long as `goimports` passes